### ISSUE TITLE
Tx Dropped without replacement

### ISSUE THREAD

**Issue by jasjyotsinghjaswal** [smartcontractkit/full-blockchain-solidity-course-py#296]
Hi ,
Sometimes my transactions fail with the below error. I understand its probably a new block getting added by the time our  transaction gets in. But is there a workaround.

Sometimes it fails due to other errors and simply works on rerunning the deploy script
![image](https://user-images.githubusercontent.com/24617050/139693303-63dc4171-109c-4590-993e-1f0aa6f4901e.png)

**Comment by cromewar** [comment]
Yes this happens frequently, you may refer to this post for more info: https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/301
